File: ECommerceApp/DataModel/Variant+CoreDataProperties.m
//
//  Variant+CoreDataProperties.m
//  ECommerceApp
//
//  Created by Prateek Raj on 11/07/18.
//  Copyright © 2018 Prateek Raj. All rights reserved.
//
//

#import "Variant+CoreDataProperties.h"

@implementation Variant (CoreDataProperties)

+ (NSFetchRequest<Variant *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"Variant"];
}

@dynamic color;
@dynamic price;
@dynamic size;
@dynamic variantId;
@dynamic products;

@end

File: ECommerceApp/DataModel/Variant+CoreDataProperties.swift
//
//  Variant+CoreDataProperties.swift
//  ECommerceApp
//
//  Created by Prateek Raj on 28/07/18.
//  Copyright © 2018 Prateek Raj. All rights reserved.
//
//

import Foundation
import CoreData


extension Variant {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<Variant> {
        return NSFetchRequest<Variant>(entityName: "Variant")
    }

    @NSManaged public var color: String?
    @NSManaged public var price: Int32
    @NSManaged public var size: Int32
    @NSManaged public var variantId: Int32
    @NSManaged public var products: Product?

}
